src/Functions/FunctionsMiscellaneous.h

```diff
@@ -50,7 +50,8 @@ class ExecutableFunctionExpression : public IExecutableFunction
          return expr_columns.getByName(signature->return_name).column;
     }
 
-bool useDefaultImplementationForNulls() const override { return false; }
+    bool useDefaultImplementationForNulls() const override { return false; }
+    bool useDefaultImplementationForNothing() const override { return false; }
```

Comment:
Not clear why this is needed.
Can you add a comment?

Reply:
Added comment and an example